Application/Client/src/app/_common/services/headers.service.ts

```diff
@@ -0,0 +1,24 @@
+import {Injectable} from '@angular/core';
+import {Headers} from '@angular/http';
+import {CookieService} from 'angular2-cookie/core';
+
+@Injectable()
+export class HeadersService {
+  private sessionObj;
+  public headers;
+  constructor( private _cookieService:CookieService) {
+  }
+
+  getHeaders(): Headers {
```

Comment:
I think this is already been written in some other file do we need to write it again?